src/completion_queue.rs

```diff
@@ -115,16 +116,17 @@ pub struct WorkCompletion {
 
 impl WorkCompletion {
     /// Get work request Id
-    pub(crate) const fn wr_id(&self) -> WorkRequestId {
+    pub const fn wr_id(&self) -> WorkRequestId {
         WorkRequestId(self.inner_wc.wr_id)
     }
 
     /// Get work completion result, if success returns length, otherwise returns error
-    pub(crate) fn err(&self) -> Result<usize, WCError> {
+    #[allow(clippy::unwrap_in_result, clippy::expect_used)]
```

Comment:
We don't use unwrap and expect in lib crate, as people are not expecting panic while using these crates.

Reply:
https://github.com/datenlord/async-rdma/pull/28/commits/97b18215ebd842124b08e823f0dd9b08f4da5da2 fixed